packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2314,6 +2314,31 @@ export class ContainerRuntime
 		return result;
 	}
 
+	/**
+	 * Returns an aliased data store with the given alias.
+	 * @param alias - The alias for the data store.
+	 * @returns - The aliased data store as a FluidObject. Throws usage error if the data store
+	 * is not aliased or if it doesn't exist.
+	 */
+	public async getAliasDataStore(alias: string): Promise<FluidObject> {
+		await this.dataStores.waitIfPendingAlias(alias);
+		const internalId = this.internalId(alias);
+		const context = await this.dataStores.getDataStore(internalId, { wait: false });
+		if (!(await context.isRoot())) {
```

Comment:
Not directly about this change, but it's strange to me that this is async.

Reply:
Yep, @andre4i and I have talked about this. It's not really possible to make this sync due to back-compat.